python/example/network_ring_mpi.py

```diff
@@ -108,7 +109,7 @@ def global_properties(self, kind):
         return self.props
 
 # (11) Instantiate recipe
-ncells = 50
+ncells = 500
```

Comment:
Why increase the number of cells?

Reply:
The tutorial text actually mentions 500 cells, but the final plot (and the code in the `.py`) used `ncells=50` to not generate an overcrowded plot. See also the caption of the plot.